src/Nri/Ui/Checkbox/V6.elm

```diff
@@ -0,0 +1,709 @@
+module Nri.Ui.Checkbox.V6 exposing
+    ( Model, Theme(..), IsSelected(..)
+    , view, viewWithLabel
+    , selectedFromBool
+    )
+
+{-|
+
+
+# Changes from V6:
+
+  - Adds `Highlighted` constructor to `Theme`.
+
+@docs Model, Theme, IsSelected
+
+@docs view, viewWithLabel
+
+@docs selectedFromBool
+
+-}
+
+import Accessibility.Styled as Html
+import Accessibility.Styled.Aria as Aria
+import Accessibility.Styled.Style
+import Accessibility.Styled.Widget as Widget
+import Css exposing (..)
+import Css.Global
+import Html.Events
+import Html.Styled
+import Html.Styled.Attributes as Attributes exposing (css)
+import Html.Styled.Events as Events
+import Json.Decode
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1 as Fonts
+import Nri.Ui.Html.V3 as HtmlExtra exposing (defaultOptions)
+import Svg
+import Svg.Attributes
+
+
+{-| -}
+type alias Model msg =
+    { identifier : String
+    , label : String
+    , setterMsg : Bool -> msg
+    , selected : IsSelected
+    , disabled : Bool
+    , theme : Theme
+    }
+
+
+{-|
+
+    = Selected --  Checked (rendered with a checkmark)
+    | NotSelected -- Not Checked (rendered blank)
+    | PartiallySelected -- Indeterminate (rendered dash)
```

Comment:
🎨 4-space indent will render as a code block.

suggested:
```
- `Selected`: Checked (rendered with a checkmark)
- `NotSelected`: Not Checked (rendered blank)
- `PartiallySelected`: Indeterminate (rendered dash)
```

Reply:
This was copied from the previous module version. I think they may have actually intended to for this to be rendered as a code block? It's like, sort of Elm code, after all. Either way, I think your suggestion is nicer to read.